src/css/Inventory.css

```diff
@@ -26,15 +26,17 @@
     cursor: pointer;
     transition: all .5s ease;
     font-family: 'Abel', sans-serif;
-    background: #3ebab4;
+    background: #7d6c54;
+    /* background: #3ebab4; */
 }
 
 /* .toggle-select:hover {
     background: #cccccc;
 } */
 
```

Comment:
Are you still planning on doing hover states? I like them as long as the hover color doesn't clash with your new palette.

Reply:
I've only got a hover on the login button for now, using the dark gold and light gold. Let's add them to the main page once we have SASS incorporated?